packages/expo-image-loader/ios/EXImageLoader/EXImageLoader.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2019-present 650 Industries. All rights reserved.
+
+#import <UMCore/UMInternalModule.h>
+#import <UMImageLoaderInterface/UMImageLoaderInterface.h>
+#import <UMReactNativeAdapter/UMBridgeModule.h>
+#import <UIKit/UIKit.h>
+
+@interface EXImageLoader : NSObject <UMBridgeModule, UMInternalModule, UMImageLoaderInterface>
```

Comment:
Sorry for not noticing that earlier, could we replace the `UMBridgeModule` with `RCTBridgeModule`? We could remove `UMBridgeModule` from the adapter since it's not used and not needed.